src/axom/primal/operators/detail/intersect_impl.cpp

```diff
@@ -330,35 +330,42 @@ inline bool checkVertex(const Point2& p1,
                         bool includeBoundary,
                         double EPS)
 {
-  if (isGpeq(twoDcross(r2, p2, q1), 0.0, includeBoundary, EPS))
+  // The tests `isGpeq(twoDcross(...))` are checking the orientation
+  // of the triangle defined by its three arguments (CCW vs CW)
+  // Note: Comments in this function refer to regions
+  // in Figure 8 of the paper: https://hal.inria.fr/inria-00072100/document
+
+  /* *INDENT-OFF* */
```

Comment:
Why do you disable the style checks here?

Reply:
I added annotations to interpret the cases w.r.t. the figure in the paper and they would not fit in 80 chars.

Here is the legend:
![image](https://user-images.githubusercontent.com/5626552/86279160-6efc4c80-bb8e-11ea-898e-7d153a94fb5b.png)

Reply:
Thanks for the `INDENT-OFF` annotations.